client/rhel/spacewalk-client-tools/src/up2date_client/up2dateErrors.py

```diff
@@ -190,6 +190,7 @@ def __repr__(self):
         return self.value
 
     def changeExplanation(self):
+        # TODO no idea
         return _("""
 Red Hat Network Classic is not supported.
```

Comment:
Maybe we shorten it and say

   Red Hat Network Classic is not supported in Uyuni.

And remove the rest. Whatever "Red Hat Network Classic" is, but we say this method is not supported :-)

Reply:
Alright :smile: